obinlog/proto/binlog.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto2";
+package obinlog;
+import "gogoproto/gogo.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+// for date and time type, string is set
+
+// for int type int64 or uint64 is set
+// for float, double, double is set
+// for decimal, string is set in human-readable now
+// for bit, bytes is set
+
+// for text and char type, string is set
+// for blob and binary type, bytes is set
+// for enum, set, uint64 is set
+// for json, bytes is set
+message Column {
+	optional bool is_null = 1 [default = false];
+
+    optional int64 int64_value = 2;
+	optional uint64 uint64_value = 3;
+    optional double double_value = 4;
+    optional bytes bytes_value = 6;
+    optional string string_value = 7;
+}
+
+message ColumnInfo {
+    optional string  name       = 1 [(gogoproto.nullable) = false];
+    // column field type in mysql
+    optional string  mysql_type = 3 [(gogoproto.nullable) = false];
+	optional bool is_primary_key = 4 [(gogoproto.nullable) = false];
```

Comment:
is_unique or is_primary?

Reply:
the reason I say that is I think we try to restore it as much as possible

Reply:
it's is_primary,  maybe we can add more info later